astropy/units/format/vounit.py

```diff
@@ -77,9 +77,7 @@ def _all_units(cls) -> tuple[dict[str, UnitBase], frozenset[str]]:
             "y", "z", "a", "f", "p", "n", "u", "m", "c", "d",
             "", "da", "h", "k", "M", "G", "T", "P", "E", "Z", "Y"
         ]  # fmt: skip
-        # While zebi and yobi are part of the standard for binary prefixes,
-        # they are not implemented here due to computation limitations
```

Comment:
I have no idea what the computation limitations were. They were not discussed at all in the pull request where this comment was added.

Reply:
If I had to guess, the author thought there would be a problem for 2**64 or greater (i.e., exceeding a 64-bit integer)

Reply:
That can't possibly have been a serious problem because when this comment was written `astropy` already supported the zetta, yotta, ronna and quetta decimal prefixes, which are all larger than 2**64.